src/blas/backends/mklcpu/fp16.hpp

```diff
@@ -25,6 +25,8 @@
 #include <limits>
 #include <type_traits>
 
+#include "onemkl/blas/detail/mklcpu/onemkl_blas_mklcpu.hpp"
```

Comment:
Why is this header file needed here now?

Reply:
Yes, here it was useless, removed it